spec/grape/api_spec.rb

```diff
@@ -313,6 +315,13 @@ def subject.enable_root_route!
         it 'with a format' do
           get '/v1/.json'
         end
+
+        context 'when array of versions provided' do
```

Comment:
The above should be in its own contect context since for example `let(:version)` isn't used here.